examples/build.gradle

```diff
@@ -18,4 +19,17 @@ subprojects {
         toolVersion = "9.3"
         configFile = rootProject.file('../config/checkstyle/checkstyle.xml')
     }
+
+    dependencies {
+        testImplementation(platform('org.junit:junit-bom:5.9.2'))
+        testImplementation 'org.junit.jupiter:junit-jupiter'
+        testImplementation 'org.junit.platform:junit-platform-suite'
+
+        testRuntimeOnly 'org.junit.jupiter:junit-jupiter-engine'
+        testRuntimeOnly 'org.junit.vintage:junit-vintage-engine'
+    }
+
+    test {
+        useJUnitPlatform()
+    }
```

Comment:
Please keep in mind [this comment](https://github.com/testcontainers/testcontainers-java/pull/6950#discussion_r1192670551) in original PR

Reply:
why this comment was mark as resolved?